packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -351,6 +351,24 @@ Expected response body:
 }
 ```
 
+### CollectionFormat_testMulti
+
+- Endpoint: `get /collectionFormat/collectionFormat/multi`
+
+This test is testing sending a multi collection format array query parameters
+
+### CollectionFormat_testCsv
+
+- Endpoint: `get /collectionFormat/collectionFormat/csv`
+
+This test is testing sending a csv collection format array query parameters
+
+### CollectionFormat_testDefault
+
+- Endpoint: `get /collectionFormat/collectionFormat/default`
+
+This test is testing sending a default collection format array query parameters
```

Comment:
is it decided that multi is the default for collection format? Just want to check because that seems to be the behavior in the tests

Reply:
I found this is the current default behavior in cadl. though I feel it's more reasonable to set the default value as csv. There's some discussion about this in Teams Channel.

Reply:
yep @lmazuel opened some issues and this does seem to be the default behavior. thanks!

Reply:
removed the default test case for now as this is confirmed by @mikekistler the current behavior is incorrect.

Reply:
will add them back when https://github.com/Azure/cadl-azure/issues/2591 is resolved.